charts/celeborn/templates/master-statefulset.yaml

```diff
@@ -71,6 +71,22 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
```

Comment:
is it fine is none children present?

Reply:
I try in local tests, setting emptyDir does not affect master startup.

Reply:
you can test the template render by `helm template`

Reply:
if there is no syntax error and works well, it's fine then

Reply:
Local test & In Kubernetes Cluster test all fine, can run in `emptyDir` type.